acknowledgements.markdown

```diff
@@ -0,0 +1,15 @@
+---
+layout: index
+title: Acknowledgements
+permalink: /acknowledgements/
+tagline: "An open-source network proxy framework for Apache Kafka"
+---
+## Acknowledgements
+
+*Thank you for supporting Kroxylicious!*
+
+<img src="{{ '/images/zulip-icon-circle.svg' | absolute_url }}" style="height: 1.5em; vertical-align: middle;">
+<span>&nbsp;<a href="https://zulip.com">https://zulip.com</a></span>
```

Comment:
Given we don't actually use zulip should we remove the reference?

@robobario is there anything we should do to shutdown/mothball the zulip instance?

Reply:
This is a holdover from the existing website ([kroxylicious.io/acknowledgements.html](https://kroxylicious.io/acknowledgements.html)). I wasn't sure if it still needed to be there or not, so I kept it in to be safe.

Reply:
I should go shutdown the instance but still think it's okay to have the thankyou here, they were nice enough to give us a license (with this quid pro quo)

Reply:
Yeah, the `thank you` is ok, maybe we should make it clear that we don't actually use the service... Just worried about people finding a zulip reference and going hunting for us there...

Reply:
But then it's weird that we're thanking them for something we don't use any more. I think we should:

1. Shutdown the instance, then
2. Remove the acknowledgement (which could be a follow-on PR).

Reply:
fair enough, I've deactivated the organisation